server/testdata/webhook-issue-comment-created-indentation.json

```diff
@@ -0,0 +1,232 @@
+            "timeoriginalestimate": null,
+            "description": "Creating an issue for unit testing",
+            "project": {
+                "self": "http://localhost:8082/rest/api/2/project/10000",
+                "id": "10000",
+                "key": "TEST",
+                "name": "Test",
+                "avatarUrls": {
+                    "48x48": "http://localhost:8082/secure/projectavatar?avatarId=10324",
+                    "24x24": "http://localhost:8082/secure/projectavatar?size=small&avatarId=10324",
+                    "16x16": "http://localhost:8082/secure/projectavatar?size=xsmall&avatarId=10324",
+                    "32x32": "http://localhost:8082/secure/projectavatar?size=medium&avatarId=10324"
+                }
+            },
+            "fixVersions": [],
+            "aggregatetimespent": null,
+            "resolution": null,
+            "timetracking": {},
+            "customfield_10106": null,
+            "attachment": [],
+            "aggregatetimeestimate": null,
+            "resolutiondate": null,
+            "workratio": -1,
+            "summary": "unit testing",
+            "lastViewed": "2020-06-27T00:44:24.621+0000",
+            "watches": {
+                "self": "http://localhost:8082/rest/api/2/issue/TEST-4/watchers",
+                "watchCount": 1,
+                "isWatching": true
+            },
+            "creator": {
+                "self": "http://localhost:8082/rest/api/2/user?username=Test",
+                "name": "Test",
+                "key": "test",
+                "emailAddress": "sample@gmail.com",
+                "avatarUrls": {
+                    "48x48": "http://localhost:8082/secure/useravatar?avatarId=10349",
+                    "24x24": "http://localhost:8082/secure/useravatar?size=small&avatarId=10349",
+                    "16x16": "http://localhost:8082/secure/useravatar?size=xsmall&avatarId=10349",
+                    "32x32": "http://localhost:8082/secure/useravatar?size=medium&avatarId=10349"
+                },
+                "displayName": "Aida Pirahmadi",
+                "active": true,
+                "timeZone": "GMT"
+            },
+            "subtasks": [],
+            "created": "2020-06-27T00:43:53.099+0000",
+            "reporter": {
+                "self": "http://localhost:8082/rest/api/2/user?username=Test",
+                "name": "Test",
+                "key": "test",
+                "emailAddress": "sample@gmail.com",
+                "avatarUrls": {
+                    "48x48": "http://localhost:8082/secure/useravatar?avatarId=10349",
+                    "24x24": "http://localhost:8082/secure/useravatar?size=small&avatarId=10349",
+                    "16x16": "http://localhost:8082/secure/useravatar?size=xsmall&avatarId=10349",
+                    "32x32": "http://localhost:8082/secure/useravatar?size=medium&avatarId=10349"
+                },
+                "displayName": "Aida Pirahmadi",
```

Comment:
There are two instances of identifying information in the testdata file. Do you mind removing them @aidapira?